language/evm/move-to-yul/tests/Arithm.move

```diff
@@ -0,0 +1,257 @@
+// Tests basic arithmetic.
+// TODO: this file is taken from the prover tests. Convert the specs here into test assertions
+//   once unit testing works. Or remove this and use existing Move tests.
+#[contract]
```

Comment:
I wonder if we should add some sort of prefix, like `#[eth_contract]` or `#[eth(contract)]` to clearly indicate that this is an ethereum-specific thing.

Reply:
Good point. We can easily change this if we decide to do so.